src/Simpleue/Locker/LockerAbstract.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Simpleue\Locker;
+
+abstract class LockerAbstract implements LockerInterface
```

Comment:
I would suggest removing "Abstract" suffix for the class, there is no need to use it in the class name.